### DISCUSSION TITLE
메모 / 차단 리스트 일괄 삭제 버튼에 2차 확인 창을 만들어 주세요.

### DISCUSSION THREAD

**Discussion by RoTuSil** [green1052/DCRefresher-Reborn#45] [Ideas]
에임 이슈로 두번이나 날려먹어서..필요할 것 같습니다.

그리고 리스트 백업/불러오기 기능은 콘솔 명령어 입력해서 하는 방법 말고
텍스트 파일 형식으로 내보내기와 불러오기를 할 수 있는 기능은 만드실 예정이 있는지 궁금합니다.

**Comment by green1052** [comment]
1. 추가해드리겠습니다.
2. 그렇게 필요하지 않을 것 같습니다.

**Reply by green1052** [reply]
https://github.com/green1052/DCRefresher-Reborn/commit/3adb87a7f373dce2912fcab7a48b45476e5e913e

**Reply by green1052** [reply]
https://github.com/green1052/DCRefresher-Reborn/commit/fbc2f05d821b9db76fcb1b8d074d298c111b33fa
